XenAdmin/Dialogs/LicenseManager/LicenseStatus.cs

```diff
@@ -368,8 +365,7 @@ public string LicenseEntitlements
                     return Messages.LICENSE_NOT_ELIGIBLE_FOR_SUPPORT;
                 }
 
-                if ((PoolLicensingModel == LicensingModel.Creedence || PoolLicensingModel == LicensingModel.Clearwater)
-                    && CurrentState == HostState.Free)
+                if (CurrentState == HostState.Free)

```

Comment:
If the enum LicensingModel acquires one more member, will this line still be correct? I presume the author meant `if (PoolLicensingModel != LicensingModel.PreClearwater)`, right?